.github/workflows/build.yml

```diff
@@ -64,8 +64,18 @@ jobs:
         env:
           CCACHE_MAXSIZE: 500M
   macos:
-    name: macos
-    runs-on: macos-12
+    strategy:
+      matrix:
+        arch: [ x86_64, arm64 ]
+        include:
+          - arch: x86_64
+            os: macos-12
+            xcode: 13.1
```

Comment:
This will be parsed as a float. While it converts back to an identical string, and so doesn't cause a problem, I think it would be neater to quote it.

Reply:
Float parsing can cause unobvious issues.  I'll stringify it.